### PR TITLE
PLAT-653: improve network bootstrap

### PR DESCRIPTION
**Pull request by AndreyBronin**
# What I did
- network launch time and stability was improved
- packets sent count and network traffic on network launch was decreased

# How I did it

- authorize request count decreased for discovery nodes
- no new Permit needed for retry bootstrap request to node with no room for candidate
- JoinAssistant balances bootstrap process giving permit with redirect to random node 
- random delay for getting permit increase chance for getting redirect to node which is not JoinAssistant

### CODE REVIEW

File: ledger-core/scripts/insolard/bootstrap_template.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ pk_shard_count: 10000
 ma_shard_count: 1000
 vesting_pulse_period: 10
 vesting_pulse_step: 10
-majority_rule: 5
+majority_rule: 7
```
Comment (sexyhamster) [inline ledger-core/scripts/insolard/bootstrap_template.yaml:18]: is it intentional?

Comment (AndreyBronin) [reply ledger-core/scripts/insolard/bootstrap_template.yaml]: it's just for testing, reverted

File: ledger-core/network/gateway/bootstrap/requester.go [hunk 2]
```diff
@@ -205,6 +248,14 @@ func (ac *requester) Bootstrap(ctx context.Context, permit *packet.Permit, candi
 		return respData, throw.New("Bootstrap got UpdateSchedule")
 	case packet.Reject:
 		return respData, throw.New("Bootstrap request rejected")
+	case packet.Retry:
+		time.Sleep(time.Second)
```
Comment (sexyhamster) [inline ledger-core/network/gateway/bootstrap/requester.go:252]: this should be configurable?

Comment (sexyhamster) [reply ledger-core/network/gateway/bootstrap/requester.go:252]: probably, would better to use synckit.Backoff here too

Comment (cyraxred) [reply ledger-core/network/gateway/bootstrap/requester.go:252]: or at least add a TODO + JIRA pls

File: ledger-core/network/gateway/base.go [hunk 3]
```diff
@@ -204,6 +211,7 @@ func (g *Base) StartConsensus(ctx context.Context) error {
 	if network.OriginIsJoinAssistant(cert) {
 		// one of the nodes has to be in consensus.ReadyNetwork state,
 		// all other nodes has to be in consensus.Joiner
+		// TODO: fix Assistant node can't join existing network
```
Comment (cyraxred) [inline ledger-core/network/gateway/base.go:214]: pls add JIRA to the backlog and mention it here

Comment (AndreyBronin) [reply ledger-core/network/gateway/base.go:220]: PLAT-694

File: ledger-core/network/gateway/bootstrap/requester.go (hunk 2 above)
Comment (AndreyBronin) [reply ledger-core/network/gateway/bootstrap/requester.go]: fixed

**Pull request merged**